docs/alt-installation.md

```diff
@@ -50,3 +50,21 @@ github.com/earthly/hello-world:main+hello | Hello, world!
 github.com/earthly/hello-world:main+hello | Target github.com/earthly/hello-world:main+hello built successfully
 =========================== SUCCESS ===========================
 ```
+
+# Removal
+
+To remove earthly, run the following commands:
+
+    # all users
+    rm -rf ~/.earthly
+    docker rm --force earthly-buildkitd
+    docker volume rm --force earthly-cache
+
+    # mac users
+    brew uninstall earthly
+
+    # linux users
+    rm /usr/local/bin/earthly
+    rm /usr/share/bash-completion/completions/earthly
+    rm /usr/local/share/zsh/site-functions/_earthly
```

Comment:
I would make clear separate sections for mac and for Linux, each with their own set of instructions.

Like

### Linux and WSL 2

```bash
rm /usr/local/bin/earthly
rm /usr/share/bash-completion/completions/earthly
rm /usr/local/share/zsh/site-functions/_earthly
rm -rf ~/.earthly
docker rm --force earthly-buildkitd
docker volume rm --force earthly-cache
```

### Mac

```bash
brew uninstall earthly
rm -rf ~/.earthly
docker rm --force earthly-buildkitd
docker volume rm --force earthly-cache
```

Reply:
that'll make it easier for users to copy/paste the commands. i'll make the change.